packages/cookbook/README.md

```diff
@@ -45,3 +45,10 @@ Example:
 ```bash
 node utils/get-state
 ```
+
+
+## Testing
```

Comment:
It seems a little inconvenient that `pnpm test` doesn't do anything to set up `localnet` but this requires a Docker image checkout and local installation of `kurtosis-cli` so it seems a little heavy for a `package.json` script. I don't think it will be a huge impediment for developers given that:
- most devs won't be running tests for this package (unless they're running them at the root)
- `localnet` is configured on the PR actions so the test results will be available
- the setup instructions here are simple with minimal dependencies